integration_tests/helpers/mock-data/invoice/random.ts

```diff
@@ -53,37 +62,49 @@ const randomCustomer = (): Customer => {
   };
 };
 
-const randomLineItem = (): LineItem => {
+export const randomLineItem = (options?: Partial<LineItem>): LineItem => {
   const priceTier = [
     { min: 0, max: 5, unitPrice: 6.5 },
     { min: 6, max: 10, unitPrice: 0.25 },
     { min: 10, unitPrice: 8.88 },
   ][Math.floor(Math.random() * 3)]; // These match fake-prices.csv
   const quantity =
+    options?.quantity ??
     priceTier.min + Math.floor(Math.random() * (priceTier.max ?? 20));
-  const subtotal = quantity * priceTier.unitPrice;
+  const unitPrice = options?.unitPrice ?? priceTier.unitPrice;
+  const subtotal = options?.subtotal ?? quantity * unitPrice;
   return {
-    description: "Verification of sentience via address checking mechanism", // should match IPV_ADDRESS_CRI_END from fake-vendor-services.csv
-    vat: 0.2 * subtotal,
+    description:
+      options?.description ??
+      "Verification of sentience via address checking mechanism", // should match IPV_ADDRESS_CRI_END from fake-vendor-services.csv
+    vat: options?.vat ?? 0.2 * subtotal,
```

Comment:
maybe overkill but could we have the option to also pass in a VAT rate that replaces 0.2 if present?